LibreHardwareMonitorLib/Hardware/Motherboard/Lpc/EC/WindowsEmbeddedControllerIO.cs

```diff
@@ -69,10 +67,10 @@ public void WriteWord(byte register, ushort value)
 
         protected virtual void Dispose(bool disposing)
         {
```

Comment:
Shouldn't you simply remove the whole dispose pattern and the finalizer?

See https://docs.microsoft.com/en-us/dotnet/standard/garbage-collection/implementing-dispose

Reply:
@Rem0o It should be fine now, although just ```IDisposable``` should be fine here as it doesn't make a lot of sense for this class. Let me remove it, give me a sec and edit 🙂

`The disposing parameter should be false when called from a finalizer, and true when called from the IDisposable.Dispose method. In other words, it is true when deterministically called and false when non-deterministically called.`

Reply:
FanControl v81 crashes on my system (ASUS C8H), but replacing the LHM dll with the one built from the LHM master makes it work. How is that possible?

Reply:
In any case the race is removed for me if I call `GC.SuppressFinalize(this);` before the `Dispose(true);`